core/rawdb/accessors_indexes_test.go

```diff
@@ -66,9 +66,13 @@ func TestLookupStorage(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			db := NewMemoryDatabase()
 
-			tx1 := types.NewTransaction(1, common.BytesToAddress([]byte{0x11}), big.NewInt(111), 1111, big.NewInt(11111), []byte{0x11, 0x11, 0x11})
-			tx2 := types.NewTransaction(2, common.BytesToAddress([]byte{0x22}), big.NewInt(222), 2222, big.NewInt(22222), []byte{0x22, 0x22, 0x22})
-			tx3 := types.NewTransaction(3, common.BytesToAddress([]byte{0x33}), big.NewInt(333), 3333, big.NewInt(33333), []byte{0x33, 0x33, 0x33})
+			sender1 := common.BytesToAddress([]byte{0x44})
```

Comment:
To confirm--this is testing basically that the transactions are correctly stored in the DB right?  If so, might be good to test those with and without an L1MsgSender.   (maybe I'm jumping the gun and those exist below)

Reply:
Yep, couldn't hurt To leave one of them `nil`